cpp/src/rolling/rolling_detail.cuh

```diff
@@ -1295,8 +1295,9 @@ std::unique_ptr<column> rolling_window_udf(column_view const& input,
                    following_window_str.c_str());
 
   cudf::jit::get_program_cache(*rolling_jit_kernel_cu_jit)
-    .get_kernel(kernel_name, {}, {{"rolling/jit/operation-udf.hpp", cuda_source}})  //
-    ->configure_1d_max_occupancy(0, 0, 0, stream.value())                           //
+    .get_kernel(
+      kernel_name, {}, {{"rolling/jit/operation-udf.hpp", cuda_source}}, {"-arch=sm_."})  //
```

Comment:
Why the empty comments at the end? Trying to influence the auto formatter?

Reply:
yep